### DISCUSSION TITLE
Depremden etkilenen gençlere abi/ablalık - mentorlük uygulaması

### DISCUSSION THREAD

**Discussion by mustafakemaltas** [acikkaynak/deprem-yardim-projesi#44] [Ideas]
Depremde evlerini, ailelerini kaybetmiş gençlere yol gösterecek, maddi olmasa bile manevi olarak yol gösterecek, destek olacak bir uygulama. Üzerine hiç düşünmeden fikri tohum haliyle ortaya atıyorum, çeşitlenip geliştirilmesi uzun soluklu olarak iyileşmemize çok yardımcı olacaktır.

**Comment by alihansencan** [comment]
Harika bir fikir. Su an biz de benzer bir konuyu konusuyoruz. Ben Backend tarafinda yardimci olabilirim.

**Comment by alihansencan** [comment]
Arkadaslar, uzun vadede depremden etkilenenlere yetkin oldugumuz konularda ders destegi veya bir kabiliyet ögretebilecegimizi düsünüyorum. Body Discussion'una da yazmistim. Yardimci olabilecek arkadaslari bekliyorum.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebinizi Legal ekibi ile görüşmemiz sonrasında reddedildi.
İletilen Not: "Legal ret, aile bakanlığı bünyesinde yürütülebilir ancak"
